hack/test-end-to-end-docker.sh

```diff
@@ -21,12 +21,32 @@ fi
 function cleanup() {
 	return_code=$?
 	os::cleanup::all "${return_code}"
+
+	# restore journald to previous form
+	os::log::info "Restoring journald limits"
+	${USE_SUDO:+sudo} mv /etc/systemd/{journald.conf.bak,journald.conf}
+	${USE_SUDO:+sudo} systemctl restart systemd-journald.service
+	# Docker has "some" problems when journald is restarted, so we need to
+	# restart docker, as well.
+	${USE_SUDO:+sudo} systemctl restart docker.service
```

Comment:
Should be fixed, I've added `os::util::ensure::system_binary_exists 'systemctl'` checks around these calls.